### PR TITLE
Add support for Safe Redirect Manger

### PR DESCRIPTION
**Pull request by jameswburke**
* Adds integration to WP-Irving for Safe Redirect Manager Plugin.

### CODE REVIEW

File: inc/endpoints/class-components-endpoint.php [hunk 1]
```diff
@@ -160,7 +200,7 @@ public function add_api_link( array $actions, \WP_Post $post ) : array {
 		$path_url = add_query_arg(
 			'path',
 			$path,
-			rest_url( 'irving/v1/components' )
+			site_url( 'wp-json/irving/v1/components' )
```
Comment (ethitter) [inline inc/endpoints/class-components-endpoint.php]: Why this change?

Comment (jameswburke) [reply inc/endpoints/class-components-endpoint.php]: `rest_url()` uses `WP_HOME` as the base of the URL, which in Irving is the app, so the URL being generated here wasn't actually pointing to the WP instance, but to Heroku.

Comment (ethitter) [reply inc/endpoints/class-components-endpoint.php]: Ah, understood. I'd add a comment noting this. 

Also, rather than hard-coding `wp-json`, this can use `rest_get_url_prefix()`.
